dear_petition/portal/etl/extract.py

```diff
@@ -4,14 +4,14 @@
 from .parsers import case_summary, dispositions, case_info, party_info
 
 
-def extract_portal_record(source):
+def extract_portal_record(source, record_id: str):
```

Comment:
Should record_id be optional for backwards compatibility?

Reply:
I can do that.